PWGMM/Tasks/multiplicityPbPb.cxx

```diff
@@ -0,0 +1,131 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+///
+/// \brief This task creates basic histograms for Pb-Pb multiplicity analysis.
+///
+/// \author hhesouno
+
+#include "Framework/runDataProcessing.h"
+#include "Framework/AnalysisTask.h"
+#include "Common/DataModel/TrackSelectionTables.h"
+#include "Framework/ASoAHelpers.h"
+
+#include "ReconstructionDataFormats/GlobalTrackID.h"
+
+using namespace o2;
+using namespace o2::framework;
+using namespace o2::framework::expressions;
+
+struct myExampleTask {
```

Comment:
Please rename the task to something more meaningful.

Reply:
Please also change the task type name in line 123 from `adaptAnalysisTask<myExampleTask>(cfgc)` to `adaptAnalysisTask< multiplicityPbPb >(cfgc)`.